### DISCUSSION TITLE
Feedback about Quick Start

### DISCUSSION THREAD

**Discussion by NickVolynkin** [tarantool/doc#2810] [Ideas]
Feedback from a fellow developer:

Про тарантул и редис, нашёл такую статью
https://medium.com/hackernoon/tarantool-vs-redis-38a4041cc4bc.
Она не раскрывает пару важных аспектов:

1. Экосистема модулей к Redis'у. Можно в пару строчек добавить всякие RedisSearch, RedisSQL
https://redis.io/modules

   У Тарантула готового я не видел, и хз, какие есть альтернативы для некоторых модулей

2. Соответственно, когда Тарантуловоды в своём чате хвалятся, что у них SQL, а у Редиса нет, меня это только больше смущает

3. Redis очень "plug'n'play"
Прописал в конфиг кластера, он запустился, можно работать.
Тарантуловский "Quick start" не очень "quick"

Наверное, поможет какое-то сравнение, вроде "в отличие от редис, у нас нужно задавать формат данных заранее, это и делает нас быстрее, блаблабла"

![изображение](https://user-images.githubusercontent.com/8015154/106887910-69fd0e00-6718-11eb-9a90-462a085ddefb.png)

И тут пугает, неужели я не могу в продакшн этот образ поднять? Что же в Dockerfile именно не так? Что именно мне нужно сделать с ним, чтобы он стал production-ready?

**Comment by pgulutzan** [comment]
Re your mention of RediSQL, which is an external module from RedBeardLabs.
This is a description of how it executes an SQL statement
https://github.com/RedBeardLab/rediSQL/blob/master/doc/docs/references.md
"Internally it transform the string into a sqlite statement using
sqlite3_prepare_v2, execute it against the database, sqlite3_step,
and finally returns the results to the client."

So apparently if you want to compare their SQL to our SQL,
it's a matter of showing that we have improvements that are not in SQLite.
That's interesting although it wouldn't belong in our manual.
